packages/NodeNameResolver/NodeNameResolver.php

```diff
@@ -181,6 +186,12 @@ public function isStringName(string $resolvedName, string $desiredName): bool
             return $desiredName === $resolvedName;
         }
 
+        foreach(self::REGEX_WILDCARD_CHARS as $char) {
+            if (str_contains($desiredName, $char)) {
+                throw new ShouldNotHappenException('Matching of regular expressions is no longer supported. Use $this->getName() and compare with e.g. str_ends_with() or str_starts_with() instead.');
```

Comment:
fnmatch should mentioned:

```suggestion
                throw new ShouldNotHappenException('Matching of regular expressions is no longer supported. Use $this->getName() and compare with e.g. str_ends_with(), str_starts_with(), or fnmatch() instead.');
```

Reply:
fnmatch example https://3v4l.org/MkqgX